jib-build-plan/src/main/java/com/google/cloud/tools/jib/api/buildplan/FileEntriesLayer.java

```diff
@@ -265,6 +326,10 @@ public FileEntriesLayer build() {
       DEFAULT_MODIFICATION_TIME_PROVIDER =
           (sourcePath, destinationPath) -> DEFAULT_MODIFICATION_TIME;
 
+  /** Provider that returns default file ownership ("0:0"). */
```

Comment:
the comment and the return value don't really line up?

Reply:
Fixed. Also updated the Build Plan Spec doc with the last commit.